### ISSUE TITLE
Crash report 7.2: ResourcesNotFoundException

### ISSUE THREAD

**Issue by mzorz** [wordpress-mobile/WordPress-Android#5795]
All on Android 4:

````
Fatal Exception: android.content.res.Resources$NotFoundException: File res/drawable-hdpi-v4/stats_widget_promo_header.png from xml type xml resource ID #0x7f020282. If the resource you are trying to use is a vector resource, you may be referencing it in an unsupported way. See AppCompatDelegate.setCompatVectorFromResourcesEnabled() for more info.
       at android.content.res.Resources.loadXmlResourceParser(Resources.java:3749)
       at android.content.res.Resources.loadXmlResourceParser(Resources.java:3696)
       at android.content.res.Resources.getXml(Resources.java:2194)
       at android.support.graphics.drawable.VectorDrawableCompat.create(VectorDrawableCompat.java:586)
       at org.wordpress.android.ui.posts.PromoDialog.onViewCreated(PromoDialog.java:61)
       at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:904)
       at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1062)
       at android.app.BackStackRecord.run(BackStackRecord.java:684)
       at android.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:1447)
       at android.app.FragmentManagerImpl$1.run(FragmentManager.java:443)
       at android.os.Handler.handleCallback(Handler.java:733)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:136)
       at android.app.ActivityThread.main(ActivityThread.java:5590)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:515)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1280)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1096)
       at dalvik.system.NativeStart.main(NativeStart.java)
Caused by java.io.FileNotFoundException: Corrupt XML binary file
       at android.content.res.AssetManager.openXmlAssetNative(AssetManager.java)
       at android.content.res.AssetManager.openXmlBlockAsset(AssetManager.java:499)
       at android.content.res.Resources.loadXmlResourceParser(Resources.java:3727)
       at android.content.res.Resources.loadXmlResourceParser(Resources.java:3696)
       at android.content.res.Resources.getXml(Resources.java:2194)
       at android.support.graphics.drawable.VectorDrawableCompat.create(VectorDrawableCompat.java:586)
       at org.wordpress.android.ui.posts.PromoDialog.onViewCreated(PromoDialog.java:61)
       at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:904)
       at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1062)
       at android.app.BackStackRecord.run(BackStackRecord.java:684)
       at android.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:1447)
       at android.app.FragmentManagerImpl$1.run(FragmentManager.java:443)
       at android.os.Handler.handleCallback(Handler.java:733)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:136)
       at android.app.ActivityThread.main(ActivityThread.java:5590)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:515)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1280)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1096)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

**Comment by mzorz** [comment]
Closed via #5801

**Comment by mzorz** [comment]
Re-appeared again in `7.3`

**Comment by daniloercoli** [comment]
This should be fixed in 7.4, I don't see any crash for PromoDialog in 7.4
